src/components/dialog/dialog.js

```diff
@@ -154,56 +169,87 @@ class Dialog extends Modal {
    * @return {Void}
    */
   get onClosing() {
-    this.window().removeEventListener('resize', this.centerDialog);
-  }
-
-  /**
-   * Returns the current window
-   *
-   * @method window
-   * @return {Object}
-   */
-  window = () => {
-    return Browser.getWindow();
+    this.appliedFixedBottom = false;
+    this.document.documentElement.classList.remove(DIALOG_OPEN_HTML_CLASS);
+    this.window.removeEventListener('resize', this.centerDialog);
+    ElementResize.removeListener(this._innerContent, this.applyFixedBottom);
   }
 
   /**
    * Centers dialog relative to window
    *
    * @method centerDialog
+   * @param {Boolean} notRender - declares that the method was called not as part of the component lifecycle
    * @return {void}
    */
-  centerDialog = () => {
+  centerDialog = (animating) => {
     const height = this._dialog.offsetHeight / 2,
-        width = this._dialog.offsetWidth / 2;
-
-    const win = this.window();
+        width = this._dialog.offsetWidth / 2,
+        win = this.window;
 
-    let midPointY = (win.innerHeight / 2) + win.pageYOffset,
-        midPointX = (win.innerWidth / 2) + win.pageXOffset;
+    let midPointY = win.innerHeight / 2,
+        midPointX = win.innerWidth / 2;
 
     midPointY -= height;
     midPointX -= width;
 
     if (midPointY < 20) {
       midPointY = 20;
-    } else if (Bowser.ios) {
-      midPointY -= win.pageYOffset;
     }
 
     if (midPointX < 20) {
       midPointX = 20;
     }
 
+    if (this._content) {
+      // apply height to content based on height of title
+      const titleHeight = this._title ? this._title.offsetHeight : '0';
+      this._content.style.height = `calc(100% - ${titleHeight}px)`;
+    }
+
     this._dialog.style.top = `${midPointY}px`;
     this._dialog.style.left = `${midPointX}px`;
+
+    if (animating === true) {
```

Comment:
I think it sometimes passes the event, which would be truthy, so i needed this check.